### DISCUSSION TITLE
Broken symbols

### DISCUSSION THREAD

**Discussion by Alkotok** [SmartlyDressedGames/Unturned-3.x-Community#3103] [Q&A] [answered]
Hey Nelson!

I have seen some already answered issues for the same problem from 2020/21, where the problem with the new font is addressed. So I have this steam guide (https://steamcommunity.com/sharedfiles/filedetails/?id=697438101) for making custom flags in the game, and with the new font you can't put multiple symbols (for example: █), on the signs/flags, which makes the entire guide useless. You probably know this issue, if I didn't misunderstood you from your answers. Anyway, what I wanted to ask is that are you going to fix this thing in the future? (Sorry for my bad english, btw.)

**Comment by SDGNelson** [comment, marked as answer]
Sorry, it depends which glyphs are supported by the font. Doing this with text is unlikely, but maybe paintable signs (in-game image editor) in the future, or placeable modded signs for the meantime.

**Reply by Alkotok** [reply]
Thank you for your answer!
